src/amo/components/FeaturedAddonReview/index.js

```diff
@@ -89,6 +89,7 @@ export class FeaturedAddonReviewBase extends React.Component<InternalProps> {
       : null;
 
     const featuredReviewCard =
+      !featuredReview &&
       errorHandler.hasError() &&
       errorHandler.capturedError.responseStatusCode === 404 ? (
```

Comment:
I may need some hint here.
`featuredReview` is loaded but the error about the not found reply is still there.
```
errors: {
  src/amo/components/FeaturedAddonReview/index.js-564315: {
    code: "ERROR_UNKNOWN",
    messages: ["Not found."],
    responseStatusCode: 404,
  }
}
```
I can't find a good place to clear that error...